services/vision/vision.go

```diff
@@ -34,7 +37,8 @@ func init() {
 	}, newCaptureAllFromCameraCollector)
 }
 
-// A Service that implements various computer vision algorithms like detection and segmentation.
+// A Service implements various computer vision algorithms like detection and segmentation.
```

Comment:
```suggestion
// The vision service implements various computer vision algorithms like detection and segmentation.
```

Reply:
This is the phrasing for most of the services. Its confusing but I think it refers to the Service interface which appears directly above and is an intentional phrasing. If we change here I should change all-- what do you think? @np